search/src/App.js

```diff
@@ -33,7 +33,7 @@ class App extends Component {
 									data.map(item => (
 										<ResultCard key={item._id}>
 											<ResultCard.Image
-												src='https://cdn.croptrust.org/wp/wp-content/uploads/2014/12/about-us.jpg'
+												src={'thumbnails/' + item.file_name}
```

Comment:
Nice, simple solution

Reply:
It's only temporary :) The filename will land up being changed, and the request will need to be handled a bit differently...